src/index.js

```diff
@@ -19,6 +22,10 @@ if (!window.hasNativeWebXRImplementation && !window.hasNativeWebVRImplementation
     CARDBOARD_UI_DISABLED: true,
     ROTATE_INSTRUCTIONS_DISABLED: true
   };
+  if (window.cordova) {
+    polyfillConfig.DPDB_URL = null;
```

Comment:
I would like this to be configurable so we can use a `file:` db rather than `null` or a remote URL. Thoughts?